modules/shortcodes/scribd.php

```diff
@@ -48,14 +48,35 @@ function scribd_shortcode_handler( $atts ) {
  * Display the shortcode.
  *
  * @param array $atts Shortcode attributes.
+ * @return string The rendered shortcode.
  */
 function scribd_shortcode_markup( $atts ) {
-	$markup = <<<EOD
-<iframe class="scribd_iframe_embed" src="//www.scribd.com/embeds/$atts[id]/content?start_page=1&view_mode=$atts[mode]&access_key=$atts[key]" data-auto-height="true" scrolling="no" id="scribd_$atts[id]" width="100%" height="500" frameborder="0"></iframe>
-<div style="font-size:10px;text-align:center;width:100%"><a href="https://www.scribd.com/doc/$atts[id]" target="_blank">View this document on Scribd</a></div>
-EOD;
+	$sandbox = class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request()
+		? 'sandbox="allow-popups allow-scripts allow-same-origin"'
+		: '';
 
-	return $markup;
+	$url = add_query_arg(
+		array(
```

Comment:
This isn't quite as clean, but this is to avoid URL-encoding `&` in the URL.

For example, with:

```php
	$url     = sprintf(
		'https://www.scribd.com/embeds/%1$d/content?start_page=1&view_mode=%2$s&access_key=%3$s',
		absint( $atts['id'] ),
		esc_attr( $atts['mode'] ),
		esc_attr( $atts['key'] )
	);
```
`$url` can equal `https://www.scribd.com/embeds/39027960/content?start_page=1&view_mode=list&access_key=key-3kaiwcjqhtipf25m8tw`

And when then when it is escaped:
``` 
esc_url( $url ) 
```
...the `&` is URL-encoded:
```html
https://www.scribd.com/embeds/39027960/content?start_page=1&#038;view_mode=list&#038;access_key=key-3kaiwcjqhtipf25m8tw
```